### DISCUSSION TITLE
Alpha 174

### DISCUSSION THREAD

**Discussion by Eugeny** [Eugeny/tabby#5974] [Release Notes]
- Fixed admin tabs not opening when debug VC libs are not installed
- Fixed acrylic accent color and dragging lag on Windows (Matheus Castello) [#5902](https://github.com/Eugeny/tabby/pull/5902)
- Fixed Sogou IME not working
- Fixed SVG and HTML icons in tab headers not showing up
- Relax OSC52 processing rules to support clipboard mgmt in tmux
- SSH: fixed .ssh/config port binding parsing
- Local: set $COLORTERM
- Locales: added Portuguese
- Fixed app category on Mac (X-0x01) [#5844](https://github.com/Eugeny/tabby/pull/5844)
- Settings: new color picker
- Sort hosts and show groups in jumplists
- SSH: fixed newlines in banners/MOTD

<hr /><em>This discussion was created from the release <a href='https://github.com/Eugeny/tabby/releases/tag/v1.0.174'>Alpha 174</a>.</em>

**Comment by Otpvondoiats** [comment]
when next
